docs/source/release-notes/2020-2-2-Asteria.md

```diff
@@ -2,9 +2,9 @@
 
 This version of Vircadia is an update to 2020.2.1. It is compatible with the 2020.2.0-Asteria protocol. This version was released on June 26th, 2020.
 
-## Interface (Codename Athena)
+## Interface (Code Name Athena)
```

Comment:
This is inconsistent will all other occurences of code-names.
According to the dictionary it is `code-name`, but since it has become a fairly normal word nowadays we could use `codename` instead.
```suggestion
## Interface (Codename Athena)
```

Reply:
Thanks. I changed one or two then noticed all the others and backed out the changes, except this one :D